sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/PlayRequest.java

```diff
@@ -37,22 +37,22 @@ public final class PlayRequest {
     private String operationContext;
 
     /**
-     * Get the playSourceInfo property: The source of the audio to be played.
+     * Get the playSources property: The source of the audio to be played.
      *
-     * @return the playSourceInfo value.
+     * @return the playSources value.
      */
-    public PlaySourceInternal getPlaySourceInfo() {
-        return this.playSourceInfo;
+    public List<PlaySourceInternal> getPlaySources() {
+        return this.playSources;
     }
 
     /**
-     * Set the playSourceInfo property: The source of the audio to be played.
+     * Set the playSources property: The source of the audio to be played.
      *
-     * @param playSourceInfo the playSourceInfo value to set.
+     * @param playSources the playSources value to set.
      * @return the PlayRequest object itself.
      */
-    public PlayRequest setPlaySourceInfo(PlaySourceInternal playSourceInfo) {
-        this.playSourceInfo = playSourceInfo;
+    public PlayRequest setPlaySources(List<PlaySourceInternal> playSources) {
```

Comment:
@atazimsft can you please point me to the overload method which takes play source as a object ?

Reply:
For your reference we need to have a overload method as we do in C# as below:
<img width="663" alt="image" src="https://github.com/Azure/azure-sdk-for-java/assets/20677492/bcad352c-9643-47a4-a3ff-83a6ec2d17ab">